components/mysql/actions/delete-row/delete-row.mjs

```diff
@@ -0,0 +1,48 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-delete-row",
+  name: "Delete Row",
+  description: "Delete an existing row. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/delete.html)",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async deleteRows({
+      table, condition, values = [],
+    }) {
+      const options = {
+        sql: `DELETE FROM \`${table}\` WHERE ${condition}`,
+        values,
+      };
+      return await this.mysql.executeQueryConnectionHandler(options);
+    },
+  },
+  props: {
+    mysql,
+    table: {
+      propDefinition: [
+        mysql,
+        "table",
+      ],
+    },
+    condition: {
+      propDefinition: [
+        mysql,
+        "whereCondition",
+      ],
+    },
+    values: {
+      propDefinition: [
+        mysql,
+        "whereValues",
+      ],
+    },
+  },
+  async run() {
+    return await this.deleteRows({
```

Comment:
I would like to suggest adding a summary in every action. You can check an example [here](https://github.com/PipedreamHQ/pipedream/pull/1865/files#diff-b8ca1aa14581a64059c7fad21406e3c0222e7e3931a28c8e2b8a14a5903edf77)